status-whisper-usage-spec.md

```diff
@@ -245,6 +246,29 @@ Public and group messages are encrypted using symmetric encryption and the key i
 
 One-to-one messages are encrypted using asymmetric encryption.
 
+## Message confirmations
+
+Sending a message is a complex process where many things can go wrong. Message confirmations tells a node that a message originating from it has been received by its peers.
+
+A node MAY send a message confirmation for any batch of messages received with a packet Messages Code (`0x01`).
+
+A message confirmation is sent using Batch Acknowledge packet (`0x0b`) or Message Response packet (`0x0c`).
```

Comment:
Whats their int packet code?

Reply:
`11` and `12`. What's not clear with hex?

Reply:
As long as we are consistent, it can lead to confusion if we aren't careful about it IMO. See e.g. https://specs.vac.dev/waku.html and packet codes